gnuradio-runtime/lib/math/random.cc

```diff
@@ -141,9 +141,12 @@ namespace gr {
   float
   random::laplacian()
   {
-    float z = ran1()-0.5;
-    if(z>0) return -logf(1-2*z);
-    else return logf(1+2*z);
+    float z = ran1();
+    if (z > 0.5){
+      return -logf(2*(1-z));
+    }else{
+      return logf(2*z);
```

Comment:
I know I can move this out of else...

Reply:
spacing around `else`, please :) I know we're about to do automated formatting in the near future, but let's not be savages until then. Or, really, omit the `else` alltogether.

Reply:
Also, this is an algorithmic change, and I can't comment on unchanged lines but: please update the copyright year.